Sources/Shared/UI/Dashboard/DashboardViewController.m

```diff
@@ -62,30 +62,23 @@ - (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibB
 
 - (void)dealloc
 {
-    [_arrDashboard release];
-	_arrDashboard = nil;
+    _arrDashboard = nil;
```

Comment:
You should also remove the ` ... = nil;` statements, I think they are not necessary.
```
You may implement a dealloc method if you need to manage resources other than releasing instance variables. You do not have to (indeed you cannot) release instance variables, but you may need to invoke [systemClassInstance setDelegate:nil] on system classes and other code that isn’t compiled using ARC.

Custom dealloc methods in ARC do not require a call to [super dealloc] (it actually results in a compiler error). The chaining to super is automated and enforced by the compiler.
```
https://developer.apple.com/library/mac/releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html#//apple_ref/doc/uid/TP40011226-CH1-SW14